doc/debug.md

```diff
@@ -21,9 +21,6 @@ class MyGame extends Game with FPSCounter {
 
 ## BaseGame features
 
-If you are using `BaseGame`, you don't have to apply the `FPSCounter` mixin to get access to the
```

Comment:
We do have the doc explaining how to add and use FPS somewhere else right?

Reply:
It is explained in the earlier section about `Game`